Tests/NatsTests/Integration/ConnectionTests.swift

```diff
@@ -367,50 +357,42 @@ class CoreNatsTests: XCTestCase {
 
     func testCredentialsAuth() async throws {
         logger.logLevel = .debug
-        let currentFile = URL(fileURLWithPath: #file)
-        // Navigate up to the Tests directory
-        let testsDir = currentFile.deletingLastPathComponent().deletingLastPathComponent()
-        // Construct the path to the resource
-        let resourceURL =
-            testsDir
-            .appendingPathComponent("Integration/Resources/jwt.conf", isDirectory: false)
-        natsServer.start(cfg: resourceURL.path)
-        print("server started with file: \(resourceURL.path)")
-
-        let credsURL = testsDir.appendingPathComponent(
-            "Integration/Resources/TestUser.creds", isDirectory: false)
+        let bundle = Bundle.module
+        natsServer.start(cfg: bundle.url(forResource: "jwt", withExtension: "conf")!.relativePath)
+
+        let creds = bundle.url(forResource: "TestUser", withExtension: "creds")!
 
         let client = NatsClientOptions().url(URL(string: natsServer.clientURL)!).credentialsFile(
-            credsURL
+            creds
         ).build()
         try await client.connect()
         let subscribe = try await client.subscribe(subject: "foo").makeAsyncIterator()
         try await client.publish("data".data(using: .utf8)!, subject: "foo")
-        let message = await subscribe.next()
-        print("message: \(message!.subject)")
+        let _ = await subscribe.next()
```

Comment:
does linter force you to have let? it doesn't matter, just curious.

Reply:
It doesn't, not sure why I had it like this, removed

Reply:
if it's returning something we have to have something on the left I thought, other was Xcode was shouting at me, but not necessarily have the `let` keyword:
```swift
_ = await subscribe.next()
```
but I'm happy if you're happy in any case.

Reply:
Ah right, Xcode complains, linter doesn't 🤷 In that case I'll change it to `_ = await subscribe.next()` to make everyone happy